src/components/common/LargeButton.js

```diff
@@ -0,0 +1,38 @@
+// 3rd party libraries - core
+import React from 'react';
+import {Button, Icon} from 'react-native-elements';
+import {
+    facebookBlueButtonTransparent, headerWhite, headerWhiteTransparent, navyBlueButtonTransparent,
+    overlayBlue
+} from "../../styles";
+import {StyleSheet} from "react-native";
+// 3rd party libraries - additional
+// styles and language
+
+
+
+const LargeButton = ({iconType, iconName, iconColor, iconSizeOverride, buttonStyleOverride, title, onPress}) => {
+    const {buttonStyle} = styles;
+
+    return (
+        <Button
+            large
+            transparent
+            icon={<Icon type={iconType} name={iconName} size={iconSizeOverride || 25} color={iconColor}/>}
```

Comment:
The `Button` component was redone in the RNE upgrade with breaking changes. The `icon` prop now expects an actual `<Icon>` element, instead of the props we were passing before.